tapir/coop/forms.py

```diff
@@ -99,6 +105,29 @@ class Meta:
 
 class DraftUserRegisterForm(forms.ModelForm):
     phone_number = TapirPhoneNumberField()
+    num_shares = forms.IntegerField(
+        label=_("Number of Shares"),
+        initial=COOP_DEFAULT_SHARES,
+        min_value=COOP_MIN_SHARES,
+        max_value=COOP_MAX_SHARES,
+        help_text=_(
+            "Number of shares you would like to purchase. The price of one share is EUR %(share_price)s. "
+            "You need to purchase at least one share to become member of the cooperative. "
+            "To support our cooperative even more, you may voluntarily purchase more shares."
+        )
+        % {"share_price": COOP_SHARE_PRICE},
+    )
+    is_investing = forms.BooleanField(
+        initial=False,
+        required=False,
+        label=_(
+            "I would like to join the membership list as an investing member (= sponsoring member)"
+        ),
+        help_text=_(
+            "<b>Note</b>: Investing members are sponsoring members. They have no voting rights in the General "
+            "Assembly and cannot use the services of the cooperative that are exclusive to ordinary members. "
+        ),
+    )
```

Comment:
I like the extra information. To have it consistent, maybe we should add it to `DraftUserForm` too

Reply:
And I created #267 in any case

Reply:
I think SuperCoop Berlin doesn't use this feature. We've added it to our login screen and added a link to our website.